corehq/ex-submodules/pillowtop/migrations/0008_sync_es_with_couch_webusers.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def sync_couch_webusers_with_es(*args, **kwargs):
+    call_command('sync_es_webusers')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pillowtop', '0007_copy_xforms_checkpoint'),
+    ]
+
+    operations = [
+        migrations.RunPython(sync_couch_webusers_with_es)
```

Comment:
I think a [`RequestReindex`](https://github.com/dimagi/commcare-hq/blob/c3e3c4d02510aa1babf3dc5bd28acae817407b29/corehq/preindex/django_migrations.py#L78) operation should be added here.

Reply:
@millerdev Thanks. I spoke to @AmitPhulera and he pointed me toward the new process of updating mappings (i.e. through migration). I subsequently created [this PR](https://github.com/dimagi/commcare-hq/pull/32886) that should be applied to production (I already confirmed it works on staging) to actually update the index with the necessary fields. Seems like the way I did it before didn't work as expected.

After that has been applied to prod I can go ahead with this PR to populate those fields with the existing data.